README.md

```diff
@@ -76,7 +81,7 @@ We have a small collection of algorithms, one for every day of the month. Scroll
 ###  **December 5 - The Tower of Hanoi**
 - **Problem**
   - Let’s slowly spice things up with a simple algorithm, yet, mind-bending if you have never seen it before. Hanoi tower has a beautiful literate solution using recursion.
-  - Say we have 3 towers. The first tower consists of a number of disks with the smallest ones at the top and the largest ones at the bottom. The second and third tower have no disks.
+  - Say we have 3 towers. The first tower consists of a number of disks with the smallest ones at the top and the largest ones at the bottom. The second and third tower has no disks.
```

Comment:
Should be "have" not "has".
"The third tower has no disks" - Correct
"The second and third tower has no disks" - Wrong